src/platform/plugins/shared/data_views/common/types.ts

```diff
@@ -502,7 +502,9 @@ export type DataViewSpec = {
    */
   version?: string;
   /**
-   * Data view title
+   * Contrary to its name, this property sets the index pattern of the data view. (e.g. `logs-*,metrics-*`)
+   *
+   * Use the `name` property instead to set a human readable name for the data view.
```

Comment:
Oh legacy code... At one point this was both the pattern and display name, but that hasn't been the case in a long time. Makes sense to call it out explicitly.